src/components/templates/Page/Page.test.tsx

```diff
@@ -9,11 +9,6 @@ describe('Page Component', () => {
         render(<Default />);
     });
 
-    it('should render a Page with a title component', () => {
```

Comment:
Removing titleComponent because we don't need Simple/Advanced switch already, so test is irrelevant too